packages/office-ui-fabric-react/gulpfile.js

```diff
@@ -16,6 +19,27 @@ let {
   packageFolder = ''
 } = buildConfig;
 
+let visualTestClean = build.subTask('visualTestClean', (gulp, options, done) => {
+  return del(['failures/*png', 'results/*png']).then(() => done());
+});
+
+let visualTest = build.subTask('visualtest', (gulp, options, done) => {
+  gulpConnect.server({
+    port: 43210,
+    livereload: false,
+    directoryListing: false
+  });
+  if (!options.args['debug']) {
+    let casperJs = require('gulp-phantomcss');
+    gulp.src(['lib/**/*.visualtest.js'])
+      .pipe(casperJs(
+        {
+          screenshots: 'visualtests',
```

Comment:
"baselines" is a more common name for this file. Visualtests make me think the tests go here, not the baseline images.